src/scss/layouts/_mainwrapper.scss

```diff
@@ -1,7 +1,8 @@
 .mainWrapper {
-  background-color: blue;
   width: 1280px;
   height: 720px;
+  margin-top: 60px;
+  border: 1px solid black;
 }
 .upperSection {
   display: flex;
```

Comment:
Dlaczego zmieniałeś pliki CSS?

Reply:
powinno to iść w innym PR, już jest dobrze